aries-site/src/data/structures/templates/templates.js

```diff
@@ -222,6 +223,29 @@ export const templates = [
       'sorting criteria',
     ],
   },
+  {
+    name: 'Selector',
+    available: true,
+    cardOrder: 4,
+    description: `A selection control that allows users to choose one or
+    more related options to access information, filter, make selections
+    or take actions.`,
```

Comment:
I would not say "take actions" because we had a lot of confusion in terms of that functionality. The selector is not a button, therefore not a place to take action.

Reply:
@vavalos5 lets work on re-wording